test/integration/default/verify/controls/aws_route_table.rb

```diff
@@ -0,0 +1,39 @@
+fixtures = {}
+[
+  'routetable_rtb_route_table_id',
+  'routetable_rtb_associations',
+  'routetable_rtb_propagating_vgws',
+  'routetable_rtb_routes',
+  'routetable_rtb_tags',
+  'routetable_rtb_vpc_id',
+].each do |fixture_name|
+  fixtures[fixture_name] = attribute(
+    fixture_name,
+    default: "default.#{fixture_name}",
+    description: 'See ../build/ec2.tf',
+  )
+end
+
+control "aws_route_table recall" do
+  describe aws_route_table(fixtures['routetable_rtb_route_table_id']) do
+    it { should exist}
+  end
+
+  describe aws_route_table do
+    it { should exist }
+  end
+
+  describe aws_route_table('rtb-2c60ec44') do
+    it { should_not exist }
+  end
+end
+
+control "aws_vpc properties" do
+  describe aws_route_table(fixtures['routetable_rtb_route_table_id']) do
+    its('vpc_id') { should eq fixtures['routetable_rtb_vpc_id'] }
+    its('tags') { should eq fixtures['routetable_rtb_tags'] }
+    its('routes') { should eq fixtures['routetable_rtb_routes']}
```

Comment:
Huh.  Does our "TF output => YAML => attribute loader => fixture hash" system preserve arrays / deep data structures?

Reply:
When I threw a puts in my control I saw a hash, I'll triple check this though.